src/core/client/admin/components/StoryInfoDrawer/StoryInfoDrawer.css

```diff
@@ -0,0 +1,12 @@
+.root {
+  position: fixed;
+  top: 0;
+  right: 0;
+
+  width: 646px;
+  height: 100%;
+
+  display: flex;
+  flex-flow: column nowrap;
+  flex-direction: column;
```

Comment:
Is it possible we could use a childed `<Flex>` control to do this? I know @cvle an @wyattjoh tend to prefer the use of `<Flex>` to inline CSS flex styling.